File: src/Entity/Commande.php
<?php

namespace App\Entity;

use App\Repository\CommandeRepository;
use Doctrine\ORM\Mapping as ORM;


#[ORM\Entity(repositoryClass: CommandeRepository::class)]
class Commande
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column]
    private ?int $id = null;

    #[ORM\Column(length: 255)]
    private ?string $nomClient = null;

    #[ORM\Column(length: 255)]
    private ?string $addresseClient = null;

    #[ORM\Column]
    private ?int $numeroClient = null;

    #[ORM\ManyToOne(inversedBy: 'commande')]
    
   
    private ?Livreur $livreur = null;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNomClient(): ?string
    {
        return $this->nomClient;
    }

    public function setNomClient(string $nomClient): static
    {
        $this->nomClient = $nomClient;

        return $this;
    }

    public function getAddresseClient(): ?string
    {
        return $this->addresseClient;
    }

    public function setAddresseClient(string $addresseClient): static
    {
        $this->addresseClient = $addresseClient;

        return $this;
    }

    public function getNumeroClient(): ?int
    {
        return $this->numeroClient;
    }

    public function setNumeroClient(int $numeroClient): static
    {
        $this->numeroClient = $numeroClient;

        return $this;
    }

    public function getLivreur(): ?Livreur
    {
        return $this->livreur;
    }

    public function setLivreur(?Livreur $livreur): static
    {
        $this->livreur = $livreur;

        return $this;
    }
}

File: src/main/java/com/example/demo4/entities/commande.java
    /*
     * To change this license header, choose License Headers in Project Properties.
     * To change this temlivreure file, choose Tools | Temlivreures
     * and open the temlivreure in the editor.
     */
    package com.example.demo4.entities;

    /**
     *
     * @author asus
     */
    public class commande extends livreur {
        private int id_commande;
        private double rating;

    private String nom_client,addresse_client,numero_client;
    public int id;
    public livreur livreur;

        public commande() {
        }

        public commande(int id_commande, int id, String nom_client, String addresse_client, String numero_client) {
            this.id_commande = id_commande;


            this.id = id;
            this.nom_client = nom_client;
            this.addresse_client = addresse_client;

            this.numero_client = numero_client;


        }
        public commande(int id ,String nom_client, String addresse_client, String numero_client, Double rating) {


            this.id = id;
            this.rating = rating;
            this.nom_client = nom_client;
            this.addresse_client = addresse_client;
            this.numero_client = numero_client;


        }

        public commande(int id_commande, int id, livreur livreur, String nom_client, String addresse_client, String numero_client) {
            this.id_commande = id_commande;


            this.id = id;
            this.livreur = livreur;
            this.nom_client = nom_client;
            this.addresse_client = addresse_client;

            this.numero_client = numero_client;


        }
        public commande(int id, String nom_client, String addresse_client, String numero_client) {



            this.id = id;

            this.nom_client = nom_client;
            this.addresse_client = addresse_client;

            this.numero_client = numero_client;


        }

        public int getId_commande() {
            return id_commande;
        }



        public double getRating() {
            return rating;
        }

        public void setRating(double rating) {
            this.rating = rating;
        }

        public int getId() {
            return id;
        }

        public livreur getlivreur() {
            return livreur;
        }

        public void setId_commande(int id_commande) {
            this.id_commande = id_commande;
        }





        public void setId(int id) {
            this.id = id;
        }
        public void setNom_client(String nom_client) {
            this.nom_client = nom_client;
        }
        public String getNom_client() {
            return nom_client;
        }


        public void setAddresse_client(String addresse_client) {
            this.addresse_client = addresse_client;
        }
        public String getAddresse_client() {
            return addresse_client;
        }
        public void setNumero_client(String numero_client) {
            this.numero_client = numero_client;
        }
        public String getNumero_client() {
            return numero_client;
        }


        public void setLivreur(livreur livreur) {
            this.livreur = livreur;
        }

        @Override
        public String toString() {
            return "commande{" + "id_commande=" + id_commande +  ", id=" + id +  ", nom_client=" + nom_client + '}';
        }







    }
